### PR TITLE
UIDATIMP-185: Create temporary MARC Bib Load option on Choose jobs pane

### PR DESCRIPTION
**Pull request by VictorSoroka1**

### CODE REVIEW

File: test/bigtest/tests/uploading-jobs-display-test.js [hunk 1]
```diff
@@ -246,4 +246,60 @@ describe('Uploading jobs display', () => {
       });
     });
   });
+
+  describe('when there are marc records exist', () => {
```
Comment (SergiyVSergiyenko) [inline test/bigtest/tests/uploading-jobs-display-test.js]: "when marc records exist" would be better

Comment (VictorSoroka1) [reply test/bigtest/tests/uploading-jobs-display-test.js]: Agree. Changed.

File: src/components/UploadingJobsDisplay/UploadingJobsDisplay.js [hunk 2]
```diff
@@ -452,6 +458,69 @@ export class UploadingJobsDisplay extends Component {
...
+    }
+  };
+
+  // TODO: this is temporary way (will be changed/removed) of deciding whether to allow to initiate
+  // the process of loading MARC BIB files (UIDATIMP-185)
+  // later the button and logic for showing will be removed
+  renderLoadMarcButton = menu => {
+    const {
+      files = [],
+      loadingRecordsInProgress,
```
Comment (YevheniiMaltsev) [inline src/components/UploadingJobsDisplay/UploadingJobsDisplay.js]: It seems, recordsLoadingInProgress sounds better

Comment (VictorSoroka1) [reply src/components/UploadingJobsDisplay/UploadingJobsDisplay.js]: Agree. Gave it a long thought and think that it may be better. Will change.

**Pull request merged**